challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        print(inp_array.shape)
</pre><p>maybe a reminder here of what the deminsions mean. What is three here for example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>